mvpoly/src/prime.rs

```diff
@@ -448,6 +448,26 @@ impl<F: PrimeField, const N: usize, const D: usize> MVPoly<F, N, D> for Dense<F,
     ) -> HashMap<usize, F> {
         unimplemented!()
     }
+
+    fn is_multilinear(&self) -> bool {
+        if self.is_zero() {
+            return true;
+        }
```

Comment:
Piggybacking on what Marc said, it seems you consider a zero polynomial to be multilinear here.

Reply:
It is.